modules/recipes_magazin/config/install/core.entity_form_display.comment.recipe_review.default.yml

```diff
@@ -1,3 +1,4 @@
+uuid: 82cd6f0d-41a2-4502-96c2-3505b939159b
```

Comment:
I know this is not a problem with the configuration in the distribution because of the core patch we have applied. However, is this also safe in a feature?